conversionSetup.sh

```diff
@@ -4,9 +4,8 @@
 # install pandoc
 sudo apt-get install pandoc
 
-# install python3.8
-sudo apt install python3.8
```

Comment:
good opportunity to talk to you about tags. We can tag this as Ubuntu 20.04 specifically since python 3 is the only version on 20.04. Alternatively later we will explore OS versioning.

Reply:
Interesting... I noticed some of this yesterday as my VMs have been running Groovy 20.10, but the image I have created uses Focal 20.04.